scripts/encrypt_external_accounts_unicode.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import sys
+
+import jwe
+
+from framework.encryption import SENSITIVE_DATA_KEY
+from framework.mongo import database
+from framework.transactions.context import TokuTransaction
+
+from website.app import init_app
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+
+def verify_external_account(document):
+    assert '_id' in document, 'ExternalAccount {} has no _id'.format(document)
+    assert 'provider' in document, 'ExternalAccount {} has no provider'.format(document['_id'])
+    assert 'oauth_key' in document, 'ExternalAccount {} has no oauth_key'.format(document['_id'])
+    assert 'oauth_secret' in document, 'ExternalAccount {} has no oauth_secret'.format(document['_id'])
+    assert 'display_name' in document, 'ExternalAccount {} has no display_name'.format(document['_id'])
+
+def encrypt_key(document, key):
+    database['externalaccount'].find_and_modify(
+        {'_id': document['_id']},
+        {'$set': {
+            key: jwe.encrypt(bytes(jwe.decrypt(document[key].encode('utf-8'), SENSITIVE_DATA_KEY)), SENSITIVE_DATA_KEY)
```

Comment:
This is the only non-import line that's different from the [original script](https://github.com/CenterForOpenScience/osf.io/blob/develop/scripts/encrypt_external_accounts.py). 

First it decrypts the stored value using the old decryption method, and then encrypts it with the new method before writing to the database.